EvidenceApi.Tests/V1/UseCase/CreateDocumentSubmissionUseCaseTests.cs

```diff
@@ -134,6 +136,25 @@ public async Task ReturnsTheCreatedDocumentSubmissionWhenRequestIsValid()
             result.DocumentType.Should().Be(docType);
         }
 
+        [Test]
+        public async Task SendsANotificationWhenNotificationEmailIsNotNull()
```

Comment:
Could be useful to have another test `DoesNotSendNotificationWhenNotificationEmailNull`

Reply:
Addressed